src/test/java/seedu/address/logic/commands/ClearAllCommandTest.java

```diff
@@ -0,0 +1,54 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.address.testutil.TypicalPersons.getTypicalAddressBook;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.AppointmentBook;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.PropertyBook;
+import seedu.address.model.UserPrefs;
+
+public class ClearAllCommandTest {
+
+    @Test
+    public void execute_emptyAppointmentBook_success() {
+        Model model = new ModelManager();
+        Model expectedModel = new ModelManager();
+
+        assertCommandSuccess(new ClearAllCommand(), model, ClearAllCommand.MESSAGE_SUCCESS,
+                expectedModel);
+    }
+
+    @Test
+    public void execute_nonEmptyAppointmentBook_success() {
+        Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+        Model expectedModel = new ModelManager(getTypicalAddressBook(), new UserPrefs());
```

Comment:
Should this two be `getTypicalAppointmentBook()` instead?

Reply:
Ahhaha, this one is not implemented yet

Reply:
I think it is?